deploy/roles/eod7/tasks/main.yaml

```diff
@@ -0,0 +1,480 @@
+ 
+#  shell: yum -y groupinstall X11 --setopt=group_package_types=mandatory,default,optional
+#  shell: yum -y groupinstall gnome-desktop x11 fonts --skip-broken
+#  shell: yum -y groupinstall "gnome-desktop"
+#  yum:
+#    name: "@gnome-desktop"
+#    state: present
+# skip_broken: yes
+  tags:
+    - eod
+
+#- name: install gnome-backgrounds
+#  yum:
+#    name: gnome-backgrounds
+#    state: present
+#  tags:
+#    - eod
+  
+- name: install gdm (gnome display manager)
+  command: yum -y install gdm
+  tags:
+    - eod
+
+#- name: install Server with GUI 
+#  yum:
+#    name: "@^Server with GUI"
+#    state: present
+#  register: serverwithgui
+#  tags:
+#    - eod
+#- debug: var="{{ serverwithgui.stdout }}"
+
+#- name: install MATE (GNOME 2) Desktop 
+#  yum:
+#    name: "@MATE Desktop"
+#    state: present
+#  register: matedesktop
+#  tags:
+#    - eod
+#- debug: var="{{ matedesktop.stdout }}"
+
+# Install EPEL moved to deploy  
+#- name: install epel-latest 
+# command: yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+#  when:
+
+- name: install XFCE4 Desktop 
+  yum:
+    name: "@xfce"
+    state: present
+  register: xfce4desktop
+  tags:
+    - eod
+#- debug: var="{{ xfce4desktop.stdout }}"
+
+- name: update libraries for MATE installation
+  command: yum -y update --enablerepo=epel-testing
+
+# MATE fix for RHEL 7.5, 7.6               
+- name: install dnf (RedHat yum4/yum replacement)
+  shell: yum install -y dnf                
+- name: apply dnf advisory fix for libgtop2 dependency
+  shell: dnf upgrade --advisory=EDORA-EPEL-2018-417249babc
+
+- name: install MATE (GNOME 2) Desktop 
+#  command: yum -y groupinstall mate-desktop
+# command: yum -y install mate-desktop mate-settings-daemon* mate-background* mate-theme*
+  command: yum -y groupinstall "MATE Desktop"
+# yum:
+#   name: "@MATE Desktop"
+#   state: present
+# register: matedesktop
+# tags:
+#   - eod
+#- debug: var="{{ matedesktop.stdout }}"
+
+#- name: install General Purpose Desktop
+- name: install Fonts
+  yum:
+    name: "@fonts"
+    state: present
+  tags:
+    - eod
+
+- name: Install Firefox
+  yum:
+    name: firefox
+    state: present
+  tags:
+    - eod
+
+- name: configure custom.conf
+  template:
+    src: custom.conf
+    dest: /etc/gdm/custom.conf
+    mode: 064
+  notify:
+    - restart gdm
+#   - restart prefdm
+  tags:
+    - eod
+
+- name: Set configuration settings for gdm defaults
+  shell: "gconftool-2 --direct --config-source xml:readwrite:/etc/gconf/gconf.xml.defaults --type {{ item.type }} --set {{ item.setting }} {{ item.value }}"
+  with_items: "{{ gconf_settings }}"
+  notify:
+    - restart gdm
+#   - restart prefdm
+  tags:
+    - eod
+
+- name: Set configuration settings for gdm mandatory
+  shell: "gconftool-2 --direct --config-source xml:readwrite:/etc/gconf/gconf.xml.mandatory --type {{ item.type }} --set {{ item.setting }} {{ item.value }}"
+  with_items: "{{ gconf_settings }}"
+  kotify:
```

Comment:
@lel99999 Typo, shouldn't this be `notify`?  And is all the gconf stuff still relevant in xfce?

Reply:
fixed.